src/renderers/webgl/WebGLObjects.js

```diff
@@ -41,7 +41,7 @@ function WebGLObjects( gl, geometries, attributes, info ) {
 
 	function dispose() {
 
-		updateList = {};
+		updateMap.clear();
```

Comment:
This line produces a runtime error since `WeakMap` does not have a `clear()` method. It should be:

```js
updateList = new WeakMap();
```

Reply:
You are right. `WeakMap` documentation suggests `clear` is obsolete. Should be fixed now.